project_task_workflow/models/project_task.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 OpenSynergy Indonesia
+# Copyright 2020 PT. Simetri Sinergi Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, api
+from openerp.tools.translate import _
+from openerp.exceptions import Warning as UserError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    @api.multi
+    def write(self, vals):
+        for record in self:
+            if record._check_allowed_stage_no_restrict():
+                if "stage_id" in vals:
```

Comment:
@mikevhe18 ini dibalik aja. if "stage_id" in vals duluan baru if record._check_allowed_stage_no_restrict()